src/main/java/com/software/modsen/drivermicroservice/controllers/CarController.java

```diff
@@ -77,15 +73,15 @@ public ResponseEntity<Car> patchCarById(@PathVariable("id") @Parameter(descripti
         return ResponseEntity.ok(carService.patchCar(id, CAR_MAPPER.fromCarPatchDtoToCar(carPatchDto)));
     }
 
-    @PostMapping("/{id}/soft-delete")
+    @DeleteMapping("/{id}")
     @Operation(
             description = "Allows to soft delete car."
     )
     public ResponseEntity<Car> softDeleteCarById(@PathVariable("id") @Parameter(description = "Car id.") long id) {
         return ResponseEntity.ok(carService.softDeleteCarById(id));
     }
 
-    @PostMapping("/{id}/soft-recovery")
+    @PostMapping("/{id}/restore")
     @Operation(
             description = "Allows to soft recovery car."
     )
```

Comment:
In most cases you won't write such "restoring" deleted items routes. Soft delete is used so that not to crash analytics and history of actions.